### PR TITLE
add executedExpectationsCount to result

### PR DESCRIPTION
**Pull request by MilanLempera**
- this change allow detect tests without expectations in reporter
 - intention in #108

### CODE REVIEW

**Comment by MilanLempera** [comment]
I signed it!

**Comment by maksimr** [comment]
@MilanLempera thanks,

I think it's good idea. How will you use this property?

**Comment by MilanLempera** [comment]
I want to use it in my [karma-json-result-reporter](https://github.com/Angular-cz/karma-json-result-reporter)
https://github.com/Angular-cz/karma-json-result-reporter/blob/master/resultConverter.js#L25

I use this reporter in our [courseWare](https://github.com/Angular-cz/courseware) project. We teach people JS and angularJS and this is our tool for presenting tutorials. Correct solution is checked by tests and json result is being sent to the browser and displayed directly in tutorials. You can see small read-only example here: https://angular-cz-courseware-examples.herokuapp.com/#/todo/02-generator-with-tests

File: package.json [hunk 1]
```diff
@@ -55,6 +55,7 @@
     "Robin Gloster <robin@loc-com.de>",
     "Sahat Yalkabov <sakhat@gmail.com>",
     "Sergey Tatarintsev <sevinf@yandex-team.ru>",
-    "Stefan Dragnev <dragnev@telerik.com>"
+    "Stefan Dragnev <dragnev@telerik.com>",
+    "Milan Lempera <milanlempera@gmail.com>"
```
Comment (dignifiedquire) [inline package.json]: Please remove this, this will be updated automatically on release.

**Comment by maksimr** [comment]
LGTM

**Comment by dignifiedquire** [comment]
Looks good to me, thanks. Left one comment, and also can you please change the commit message to follow our [convention](http://karma-runner.github.io/0.13/dev/git-commit-msg.html)

**Comment by MilanLempera** [comment]
I changed my code by comments, what is next step?

**Comment by dignifiedquire** [comment]
Thanks :octocat: is happy now :)

**Pull request merged**